2-collectors/scala-stream-collector/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/metrics/MetricsDirectives.scala

```diff
@@ -0,0 +1,23 @@
+package com.snowplowanalytics.snowplow.collectors.scalastream.metrics
+
+import java.time.{Duration, LocalDateTime}
+
+import akka.http.scaladsl.server.Directive0
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.directives.BasicDirectives.extractRequestContext
+
+trait MetricsDirectives {
```

Comment:
also if you could add Scaladoc to the different public members that'd be much appreciated :+1:

Reply:
added some info